pkg/cmd/testhelpers/cmd_test_helpers.go

```diff
@@ -151,35 +151,28 @@ func CreateTestJxHomeDir() (string, string, error) {
 	if err != nil {
 		return "", "", err
```

Comment:
probably good to wrap the error and add a custom error string. Makes it easier to trace in case this error gets ever triggered.